File: projekatWP_bar/glass.js
export class Glass{
    constructor(a,b,c,d){
        this.Position=a;
        this.DrinkName=b;
        this.Price=c;
        this.Color=d;
    }

    updateGlass(a,b,c){
        this.DrinkName+=", "+a;
        this.Price+=b;
        this.Color=c;
    }
    newGl(a,b,c){
        this.DrinkName=a;
        this.Price=b;
        this.Color=c;
    }
    get color() {
        return this.Color;
    }
    get name(){
        return this.DrinkName;
    }
    get price(){
        return this.Price;
    }
    get position(){
        return this.Position;
    }
}

File: projekatWP_bar/Model/Glass.cs
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;
using System.Text.Json.Serialization;

namespace projekatWP_bar.Model{
    [Table("Glass")]
    public class Glass{
        [Key]
        [Column("ID")]
        public int ID { get; set; }
        [Column("Position")]
        public int Position { get; set; }
        [Column("DrinkName")]
        [MaxLength(255)]
        public string DrinkName { get; set; }
        [Column("Price")]
        public int Price { get; set; }
        [Column("Color")]
        public string Color { get; set; }

        [JsonIgnore]
        public Order Order { get; set; }
    }
}